data/json/recipes/armor/arms.json

```
@@ -94,7 +107,11 @@
     "book_learn": [ [ "textbook_armwest", 6 ] ],
     "using": [ [ "blacksmithing_standard", 24 ], [ "steel_standard", 2 ], [ "tailoring_leather", 2 ] ],
     "qualities": [ { "id": "CHISEL", "level": 3 } ],
-    "tools": [ [ [ "swage", -1 ] ] ]
+    "tools": [ [ [ "swage", -1 ] ] ],
+    "proficiencies": [
+      { "proficiency": "prof_armorsmithing", "required": false, "time_multiplier": 3 },
+      { "proficiency": "prof_leatherworking", "required": false, "time_multiplier": 1 }
```

A time multiplier of 1 means having the proficiency won't make any difference to time. It still makes a difference to failure so you could leave it that way, but I usually put minor proficiencies at 1.2 or 1.5 or so, and add a learning time multiplier so it doesn't count for as much xp.

```suggestion
      { "proficiency": "prof_leatherworking", "required": false, "time_multiplier": 1.5, "learning_time_multiplier": 0.5 }
```